domains/jake.json

```
@@ -1,11 +1,8 @@
 {
-  "description": "Jake's personal website and blog.",
-  "repo": "https://github.com/j-eo/j-eo.github.io",
   "owner": {
```

Email key is needed, even tho you dont want it.
Just do this
```suggestion
  "owner": {
  "email":""
  "username:"xzrci"
  }
```